vipps-checkout-api.md

```diff
@@ -96,7 +99,7 @@ The merchant defines a _dynamic options callback URL_, which is called by Vipps
 
 #### Pickup points
 
-We currently provide support for Posten/Bring, PostNord, and Instabox pickup points. As of now, we do not support lockers etc. in Posten/Bring and Postnord.
+We currently provide support for Posten/Bring, PostNord, Instabox and Helthjem pickup points. As of now, we do not support lockers etc. in Posten/Bring and Postnord.
 To enable pickup points for a logistics option, the isPickupPoint flag must be set to true.
```

Comment:
Is pickup point set in the same way for Postnord/Bring and Instabox/Helthjem?

Reply:
yeah it works the same way, seen from the user, if that's what you mean?